predict.py

```diff
@@ -51,7 +54,7 @@ def main(cfg: DictConfig) -> None:
                 # extract original index
                 orig_filename = fill_placeholders(to_absolute_path(f'{cfg["orig_path"]}/{cfg["orig_filename_template"]}'), {'{sample_name}': sample_name})
                 with uproot.open(orig_filename) as f:
-                    t = f['TauCheck']
+                    t = f[cfg["input_tree_name"]]
                     orig_index = t.arrays(['evt', 'run'], library='pd')
                     orig_index = list(map(tuple, orig_index.values))
                 
```

Comment:
Was it sufficient to solve the type problem with only converting the type at the preprocessing step (skipping the alignment step below)?